scripts/integration_test.sh

```diff
@@ -17,14 +17,29 @@
 # Setup environment for CI.
 set -e
 
-dir=`dirname $0`
+readonly dir=`dirname $0`
+
+GetStatusOfContainerBuild () {
```

Comment:
Can we just rely on the exit code of the script instead? I believe it's 0 when pass, and non-zero when failed.